geetools/ee_image_collection.py

```diff
@@ -896,6 +896,25 @@ def groupInterval(self, unit: str = "month", duration: int = 1) -> ee.List:
             lambda dr: ic.filterDate(ee.DateRange(dr).start(), ee.DateRange(dr).end())
         )
 
+        sizeName = "__geetools_generated_size__"
```

Comment:
Can you move this "static" parameter at the very start of the method ? with a small comment "# set generated properties name" (that will be used in other methods of the lib in the future).